GPU/GLES/FramebufferManagerGLES.cpp

```diff
@@ -1220,7 +955,7 @@ bool FramebufferManagerGLES::CreateDownloadTempBuffer(VirtualFramebuffer *nvfb)
 
 	draw_->BindFramebufferAsRenderTarget(nvfb->fbo);
 	ClearBuffer();
-	glDisable(GL_DITHER);
+	glDisable(GL_DITHER);  // Weird place to do this
```

Comment:
I assume it's here because downloading from a 16 bit buffer with dithering on can cause issues (for example in Danganronpa), but it probably more belongs at the callers of `FindDownloadTempBuffer`.  I think this affected AMD and Adreno both, but in different ways.

-[Unknown]